src/cram.hpp

```diff
@@ -0,0 +1,41 @@
+#ifdef PYNE_CRAM_SOLVE_C
+#error "Both cram.h (the C header for CRAM) and cram.hpp (C++) have been included, only one is allowed!"
+#endif
+
+#ifndef PYNE_CRAM_SOLVE_CPP
+#define PYNE_CRAM_SOLVE_CPP
+
+
+typedef struct pyne_cram_transmute_info_tag {
+  int n;
+  int nnz;
+  int* i;
+  int* j;
+  char** nucs;
+  int* nucids;
+  double* decay_matrix;
+} pyne_cram_transmute_info_t;
+
+extern pyne_cram_transmute_info_t pyne_cram_transmute_info;
+
+int pyne_cram_transmute_ij(int i, int j);
+
+int pyne_cram_transmute_nucid_to_i(int nucid);
+
+
```

Comment:
Is the double new line here a style choice?

Reply:
Whoops thanks for noticing this!